packages/service-library/src/servicelib/deferred_tasks/_deferred_manager.py

```diff
@@ -134,7 +134,9 @@ def __init__(
             ClassUniqueReference, type[BaseDeferredHandler]
         ] = {}
 
-        self.broker: RabbitBroker = RabbitBroker(rabbit_settings.dsn)
+        self.broker: RabbitBroker = RabbitBroker(
+            rabbit_settings.dsn, log_level=logging.DEBUG
```

Comment:
Yesterday, I had a discussion with @matusdrobuliak66 about implementing a more flexible logging system that would give us greater control over logging behavior. This would include features like adjusting the log level for specific loggers or applying filters to particular loggers.

Such functionality can be very useful, whether we need to increase log verbosity to troubleshoot issues or reduce the number of logs from a particularly noisy logger.

Can give you some details on our thoughts offline.

Reply:
I think this is a different subject, and goes with how we use the faststream library, as this specific part was not checked.
The way of reducing the log level of 3rd party does not work in this case.

Reply:
in the faststream documentation, they also talk about using StructLog which appears to be a nice log alternative as well